pkgs/development/tools/erlang/hex2nix/default.nix

```diff
@@ -30,6 +30,20 @@ buildRebar3 rec {
 
     enableDebugInfo = true;
 
+    # hex2nix does not specify version constraints, in which case
```

Comment:
Maybe it's better to submit these changes directly to `hex2nix` repo? It seems like the right thing to do.

Reply:
That does actually make sense, I'll prepare a change. Currently JSX is imported manually at version 2.8.2 in the derivation for hex2nix, do you know if there's a reason for it? The latest packaged version is 2.8.0 and I'd prefer to stick to that until the packages are regenerated.

Reply:
Cursory test of hex2nix compiled with JSX 2.8.0 shows that it still works. The changelog for JSX contains nothing that looks relevant between 2.8.0 and 2.8.2.

Reply:
Change submitted here: https://github.com/erlang-nix/hex2nix/pull/22